src/System.Management.Automation/utils/Verbs.cs

```diff
@@ -1123,34 +1123,24 @@ public class VerbInfo
         /// <summary>
         /// The verb name, used to begin command names.
         /// </summary>
-        public string Verb
-        {
-            get; set;
-        }
+        public string Verb { get; internal set; }
 
         /// <summary>
         /// The alias prefix, recommended for aliases to commands that begin with this verb.
         /// </summary>
-        public string AliasPrefix
-        {
-            get; set;
-        }
+        public string AliasPrefix { get; internal set; }
```

Comment:
Why not 
```suggestion
        public string AliasPrefix { get; init set; }
```
if the class looks like `immutable`?